Mathlib/MeasureTheory/Measure/Lebesgue/Basic.lean

```diff
@@ -577,6 +577,68 @@ theorem volume_regionBetween_eq_lintegral [SigmaFinite μ] (hf : AEMeasurable f
           (regionBetween_subset (AEMeasurable.mk f hf) (AEMeasurable.mk g hg) s)).symm
 #align volume_region_between_eq_lintegral volume_regionBetween_eq_lintegral
 
+lemma nullMeasurableSet_fst_mem {α β : Type*} [MeasurableSpace α] [MeasurableSpace β]
+    {μ : Measure α} {ν : Measure β} [SigmaFinite ν] {s : Set α} (s_mble : NullMeasurableSet s μ) :
+    NullMeasurableSet {p : α × β | p.fst ∈ s} (μ.prod ν) := by
+  convert s_mble.prod (ν := ν) nullMeasurableSet_univ
+  ext p
+  simp only [mem_setOf_eq, mem_prod, mem_univ, and_true]
```

Comment:
It's definitely not the right place for this lemma. However, I would just erase it completely, as it has a one-line proof that you can use instead in your application: `s_mble.preimage quasiMeasurePreserving_fst`.

Reply:
Thank you for the concise proof! I erased the lemma.